vendor/deps/cli-kit/REVISION

```diff
@@ -0,0 +1 @@
+fc25a17dff9d3130bb705659c002282ef7827932
```

Comment:
What is this for? Just curious

Reply:
I think it just keeps track of the git SHA of the dependency